jdaviz/utils.py

```diff
@@ -560,8 +560,16 @@ def _eqv_flux_to_sb_pixel():
                   u.ct,
                   u.DN,
                   u.DN / u.s]
-    return [(flux_unit, flux_unit / PIX2, lambda x: x, lambda x: x)
-            for flux_unit in flux_units]
+    equivs = [(flux_unit, flux_unit / PIX2, lambda x: x, lambda x: x)
+              for flux_unit in flux_units]
+
+    # We also need to convert between spaxel and pixel squared
+    equivs += [(flux_unit / u.Unit('spaxel'), flux_unit / PIX2,
+               lambda x: x, lambda x: x) for flux_unit in flux_units]
+
+    print(flux_units[1] / u.Unit('spaxel'))
+
```

Comment:
```suggestion
```

Reply:
Ha, I knew I had one left and couldn't find it (didn't just look at the diff here for some reason). Thanks.